os_dep.c

```diff
@@ -864,6 +864,18 @@ GC_INNER size_t GC_page_size = 0;
 # define HAVE_GET_STACK_BASE
 #endif /* OS2 */
 
+#ifdef QNX
```

Comment:
You can define `HAVE_GET_STACK_BASE` and `GC_get_stack_base` in the platform code instead of needing a special case here - I'll share an example in the IL2CPP pr.